api/store/mongo/migrations/main.go

```diff
@@ -32,6 +32,7 @@ func GenerateMigrations() []migrate.Migration {
 		migration_20,
 		migration_21,
 		migration_22,
+		migration_23,
```

Comment:
There is already a 23 migration, it'll need to be moved to 24 or 25.

Reply:
Yes, I will when I liberate the PR. Because it can go into some other migration, so not to change more than once, I ended up not modifying it yet.